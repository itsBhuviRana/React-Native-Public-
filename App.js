import React, { Component } from 'react'
import { Text, View } from 'react-native'

export class App extends Component {
  render() {
    return (
      <View>
        <Text> Hello rutuja!11 </Text>
      </View>
    )
  }
}

export default App
