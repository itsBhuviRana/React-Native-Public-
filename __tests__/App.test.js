/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../App';

import {test} from '@jest/globals';

//to render any other UI 
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const snapshot= renderer.create(<App />).toJSON;
  console.log("abc",snapshot)
  expect(snapshot).toMatchSnapshot();
});
