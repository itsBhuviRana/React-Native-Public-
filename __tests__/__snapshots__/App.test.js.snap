// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text>
     Hello rutuja!11 
  </Text>
</View>
`;
